docs/ota-client-guide/modules/ROOT/pages/customise-targets-metadata.adoc

```diff
@@ -9,35 +9,45 @@ endif::[]
 
 include::partial$aktualizr-version.adoc[]
 
-In some cases, you might find it useful to include extra metadata about your software inside the signed Uptane metadata that OTA Connect delivers to your device. Some reasons you might want to do this include:
+You may need to include extra metadata about your software inside the signed Uptane metadata that OTA Connect delivers to your device. Some reasons you might want to do this include:
```

Comment:
```suggestion
You may want to include extra metadata about your software inside the signed Uptane metadata that OTA Connect delivers to your device. Some reasons you might want to do this include:
```

There's no reason, in the software we provide, for a person to _need_ their own custom metadata, so let's stick with "want". Anyone who actually does need it in their own use case already knows it.

Reply:
Changed.